### ISSUE TITLE
Update a few links from GitHub links to website internal links?

### ISSUE THREAD

**Issue by brittag** [kennethreitz/python-guide#656]
Currently the installation guides for [Windows](http://docs.python-guide.org/en/latest/starting/install/win/), [OS X](http://docs.python-guide.org/en/latest/starting/install/osx/), and [Linux](http://docs.python-guide.org/en/latest/starting/install/linux/) have this at the bottom:

> To start using and see more information: [Virtual Environments](http://github.com/kennethreitz/python-guide/blob/master/docs/dev/virtualenvs.rst) docs.

It seems helpful to replace that GitHub link (http://github.com/kennethreitz/python-guide/blob/master/docs/dev/virtualenvs.rst) with the nicely formatted website version of that document (http://docs.python-guide.org/en/latest/dev/virtualenvs/). I could make that change if it's a good idea (it's a simple change), but I'm wondering if there's a good reason why it's like that.

**Comment by sigmavirus24** [comment]
There's almost certainly no good reason why it is like that. Please send a pull request (if you want to) to fix that. Thanks @brittag!

**Comment by brittag** [comment]
OK, I figured out how to fix this. In case it's helpful to see my process, to improve other documentation: first, I looked for other internal links and saw they were formatted like this:

> :ref:`virtualenvwrapper <virtualenvwrapper-ref>`

I wasn't sure how that worked, but I followed the path from the [CONTRIBUTING.md file](https://github.com/kennethreitz/python-guide/blob/master/CONTRIBUTING.md) to reading the style guide, [which reinforced](http://docs.python-guide.org/en/latest/notes/styleguide/#linking-to-sections-in-the-guide) that this is the correct format.

I still wasn't sure how to find the correct text to put inside the angle brackets to link to the Virtual Environments page, but I looked at the top of the OS X page and saw it had the ".. _some-section-ref:" text at the top, so I added ".. _virtualenvironments-ref:" at the top of the Virtual Environments page and tried making links to ":ref:`Virtual Environments <virtualenvironments-ref>`".

Then I wanted to run a local copy of the guide to test this. I didn't see instructions for doing that, so I looked at the directory, and I saw a make file, so I ran make, and it told me I needed to install Sphinx. I installed it and ran make again, and it generated some flat files - so I figured out I could test it by making changes, running make, and looking at the files. Seems to work. Yay!
